hibernate-core/src/main/java/org/hibernate/engine/jdbc/spi/SqlStatementLogger.java

```diff
@@ -120,14 +133,16 @@ public void logStatement(String statement) {
 	 */
 	@AllowSysOut
 	public void logStatement(String statement, Formatter formatter) {
-		if ( format ) {
-			if ( logToStdout || LOG.isDebugEnabled() ) {
-				statement = formatter.format( statement );
-			}
+		if ( format
+				&& ( logToStdout || LOG.isDebugEnabled() ) ) {
+			statement = formatter.format( statement );
 		}
 		LOG.debug( statement );
 		if ( logToStdout ) {
-			System.out.println( "Hibernate: " + statement );
+			if ( highlight ) {
+				statement = FormatStyle.HIGHLIGHT.getFormatter().format( statement );
+			}
+			System.out.println( "\u001b[35m[Hibernate]\u001b[0m " + statement );
```

Comment:
If not highlight you should not print Ansi escapes.

Reply:
Fixed.